cmd/skaffold/skaffold.go

```diff
@@ -33,6 +33,7 @@ type ExitCoder interface {
 }
 
 func main() {
+	instrumentation.SetOnlineStatus()
```

Comment:
IMHO we should be pinging a server when the user has disable telemetry.

Reply:
Good point. I'll add a conditional for this